src/pyatsimagebuilder/builder.py

```diff
@@ -26,6 +26,8 @@
     r'.*job.*\.py',
 ]
 ENV_PATTERN = re.compile(r'(%ENV{ *([0-9a-zA-Z\_]+) *})')
+IMAGE_BUILD_SUCCESSUL = \
+    re.compile(r' *Successfully built (?P<image_id>[a-z0-9]{12})')
```

Comment:
```suggestion
    re.compile(r' *Successfully built (?P<image_id>[a-z0-9]{12}) *$')
```
By matching the line end, it ensures the output from pip building is not also grabbed:
![image](https://user-images.githubusercontent.com/31291038/107290926-5c57c900-6a35-11eb-8439-429f4710b155.png)
Only the dash in `aiohttp-swagger` prevents this from being grabbed as the image ID.